var/spack/repos/builtin/packages/mold/package.py

```diff
@@ -10,11 +10,20 @@ class Mold(CMakePackage):
     """mold: A Modern Linker"""
 
     homepage = "https://github.com/rui314/mold"
-    url = "https://github.com/rui314/mold/archive/refs/heads/release/1.7.1.zip"
+    url = "https://github.com/rui314/mold/archive/refs/tags/v1.11.0.tar.gz"
 
     maintainers("jabcross")
 
+    version("1.11.0", sha256="99318eced81b09a77e4c657011076cc8ec3d4b6867bd324b8677974545bc4d6f")
     version("1.7.1", sha256="e155f647c4c8555697f2d9544ba2f93a67023c4fff21b5b56aa3157700e14364")
```

Comment:
Changed to `.tar.gz`:
```suggestion
    version("1.7.1", sha256="fa2558664db79a1e20f09162578632fa856b3cde966fbcb23084c352b827dfa9.")
```